packages/mjml-button/src/index.js

```diff
@@ -51,85 +92,116 @@ class Button extends Component {
   styles = this.getStyles()
 
   getStyles () {
-    const { mjAttribute, defaultUnit } = this.props
+    const {mjAttribute} = this.props,
+      width = mjAttribute('width'),
```

Comment:
I'm not sure if this is a bad practice or not, but when adding more and more variable it becomes less and less readable.

I don't know, non clickable padding allow you to select the text inside a button more easily ex : http://codepen.io/anon/pen/peRBzW